packages/blocks/src/page-block/doc/doc-page-block.ts

```diff
@@ -187,6 +207,13 @@ export class DocPageBlockComponent extends BlockElement<
     );
   };
 
+  appendParagraph = () => {
```

Comment:
Note sure where is the use case of this API?

Reply:
This is intened to used for Affine side.
To complete clicking end area of editor to append paragraph.
I found that because editor is composed in affine, it is convenient that the feature is implementd in affine.

Reply:
In this case, think the API set should be exposed on page service.

Reply:
@regischen 
"To complete clicking end area of editor to append paragraph." - can this be done in blocksuite itself ?

Reply:
Done